scripts/embedded-docs-cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
who runs this script? It's in the kairos-agent side?

Reply:
will need to be run in the package of the kairos-agent where we embed the docs